src/components/EpisodeCard/EpisodeCard.js

```diff
@@ -0,0 +1,97 @@
+import LazyLoad from 'react-lazyload'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+EpisodeCard.propTypes = {
+  episode: PropTypes.object.isRequired,
+}
+
+export default function EpisodeCard({ episode }) {
+  const { datePublished, duration } = episode
+  const months = [
+    'Jan.',
+    'Feb.',
+    'Mar.',
+    'Apr.',
+    'May.',
+    'Jun.',
+    'Jul.',
+    'Aug.',
+    'Sep.',
+    'Oct.',
+    'Nov.',
+    'Dec.',
+  ]
+  const date = new Date(datePublished * 1000)
+  const time = Math.floor(duration / 60)
+
+  return (
+    <Card>
+      <ImageAndTitleContainer>
+        <LazyLoad height={48} width={48} offset={96} overflow={true} once>
+          <ImageContainer>
+            <Image
+              src={episode.feedImage}
+              alt={'Poster'}
```

Comment:
Is that an alt-attribute that makes sense to the user? I can't figure out completely what this alt says to me but maybe it makes perfectly sense in the context.

Reply:
Yes thank you i will fix that.